docs: clean up the troubleshooting for mac users section (#64)
0xPolygon/kurtosis-cdk: docs/quickstart/deploy-stack.md

- A Linux-based OS (e.g., Ubuntu Server 22.04 LTS).
- At least 8GB RAM with a 2-core CPU.
- An AMD64 architecture system.

!!! tip

>     Ensure to install [Docker Engine](https://docs.docker.com/engine/) version 4.27 or higher to be able to run the zkEVM Prover on macOS.

Reviewer: ```suggestion
    Make sure you install [Docker Engine](https://docs.docker.com/engine/) version 4.27 or higher to run the zkEVM prover on MacOS.
```

    kurtosis run --enclave cdk-v1 --args-file params.yml --image-download always .
    ```

>     This command takes a few minutes to complete and steps up and runs an entire local CDK deployment.
> 
> When everything is set up and running, we can play around with the test CDK.

Reviewer: ```suggestion
    This command takes a few minutes to complete and sets up and runs an entire local CDK deployment. When everything is set up and running, we can play around with the test CDK.
```
